x-pack/docs/en/security/securing-communications/security-basic-setup-https.asciidoc

```diff
@@ -0,0 +1,595 @@
+and visualizations. Create the reader role to assign proper privileges to these
+users.
+
+1. Create the reader role:
+
+   a. Enter **metricbeat_reader** as the role name.
+
+   b. On the **metricbeat-\*** indices, choose the **read** privilege.
+
+   c. Under **Kibana**, click **Add Kibana privilege**.
+
+   - Under **Spaces**, choose **Default**.
+
+   - Choose **Read** or **All** for Discover, Visualize, Dashboard, and Metrics.
+
+2. Create the reader user:
+
+   a. Enter **metricbeat_reader** as the user name.
+
+   b. Enter the username, password, and other user details.
+
+   c. Assign the following roles to the **metricbeat_reader** user:
++
+[cols="1,1"]
+|===
+| Role                          | Purpose
+
+| `metricbeat_reader` | Read {metricbeat} data.
+| `monitoring_user`   | Allow users to monitor the health of {metricbeat}
+itself. Only assign this role to users who manage {metricbeat}
+| `beats_admin`       | Create and manage configurations in {beats} central
+management. Only assign this role to users who need to use {beats} central
+management.
+|===
+
+**Next**: <<configure-metricbeat-tls,Configure {metricbeat} to use TLS>>
+
+[discrete]
+[[configure-metricbeat-tls]]
+===== Configure {metricbeat} to use TLS
+
+Before starting {metricbeat}, you configure the connections to {es} and
+Kibana. You can configure authentication to send data to your secured cluster
+using basic authentication, API key authentication, or Public Key
+Infrastructure (PKI) certificates.
+
+The following instructions use the credentials for the `metricbeat_writer`
+and `metricbeat_setup` roles that you created. If you need a greater level of
+security, we recommend using PKI certificates.
+
+After configuring connections to Elasticsearch and Kibana, you'll enable the
+`elasticsearch-xpack` module and configure that module to use HTTPS.
+
+WARNING: In production environments, we strongly recommend using a separate
+cluster (referred to as the monitoring cluster) to store your data. Using a
+separate monitoring cluster prevents production cluster outages from impacting
+your ability to access your monitoring data. It also prevents monitoring
+activities from impacting the performance of your production cluster.
+
+. From the directory where you installed Elasticsearch, navigate to the
+`/kibana` directory that you created when <<encrypt-http-communication,encrypting HTTP client communications for {es}>>.
+
+. Copy the `elasticsearch-ca.pem` certificate to the directory where you
+installed Metricbeat.
+
+. Open the `metricbeat.yml` configuration file and configure the connection
+to Elasticsearch.
++
+Under `output.elasticsearch`, specify the following fields:
++
+[source,yaml]
+----
+output.elasticsearch:
+ hosts: ["<your_elasticsearch_host>:9200"]
+ protocol: "https"
+ username: "metricbeat_writer"
+ password: "<password>"
+ ssl:
+   certificate_authorities: ["elasticsearch-ca.pem"]
+   verification_mode: "certificate"
```

Comment:
Why `verification_mode: "certificate"` here? Assuming they followed the steps up until here, the certificate ES is using has the correct hostname

Reply:
@jkakavas, is the verification mode unnecessary then?